github-actions/update-version-tags/action.yml

```
@@ -0,0 +1,13 @@
+name: Update version tags
+description: Update version tags.
+
+inputs:
+  repo-token:
+    description: The GITHUB_TOKEN secret.
+
+  sha:
+    description: The SHA1 of commit to set version tags. By default, uses the release branch or commit.
+
+runs:
+  using: node16
+  main: build/index.js
```

:grey_question:  Related to https://github.com/woocommerce/grow/pull/12#issuecomment-1153858821, can we make it?
```suggestion
  main: update-version-tags.js
```

---

Changed in https://github.com/woocommerce/grow/pull/12/commits/3cbffa88e87097a6339c50115cdc648939dc4d57.